Error from docker build

Following the [Getting Started](https://docs.theairlab.org/docs/getting_started/) instructions, when running the `docker compose build` step, I get the error below from step
```
[isaac-sim 13/17] RUN cd /tmp/ &&     curl -L -O https://github.com/isaac-sim/IsaacSim-ros_workspaces/archive/main.zip &&     unzip main.zip &&     mv IsaacSim-ros_workspaces-main/humble_ws /humble_ws &&     cd /humble_ws &&     . /opt/ros/humble/setup.sh &&     colcon build --symlink-install &&     rm -rf /tmp/IsaacSim-ros_workspaces-main
```
Error:
```
1.959 Starting >>> carter_navigation
1.966 Starting >>> cmdvel_to_ackermann
1.969 Starting >>> custom_message
1.972 Starting >>> isaac_ros2_messages
1.975 Starting >>> isaac_ros_navigation_goal
1.980 Starting >>> isaac_tutorials
1.990 Starting >>> isaacsim
1.994 Starting >>> iw_hub_navigation
2.709 --- stderr: cmdvel_to_ackermann
2.709 CMake Error at CMakeLists.txt:6 (find_package):
2.709   By not providing "Findackermann_msgs.cmake" in CMAKE_MODULE_PATH this
2.709   project has asked CMake to find a package configuration file provided by
2.709   "ackermann_msgs", but CMake did not find one.
2.709 
2.709   Could not find a package configuration file provided by "ackermann_msgs"
2.709   with any of the following names:
2.709 
2.709     ackermann_msgsConfig.cmake
2.709     ackermann_msgs-config.cmake
2.709 
2.709   Add the installation prefix of "ackermann_msgs" to CMAKE_PREFIX_PATH or set
2.709   "ackermann_msgs_DIR" to a directory containing one of the above files.  If
2.709   "ackermann_msgs" provides a separate development package or SDK, be sure it
2.709   has been installed.
2.709 
2.709 
2.709 ---
2.709 Failed   <<< cmdvel_to_ackermann [0.74s, exited with code 1]
2.744 Aborted  <<< carter_navigation [0.79s]
2.871 Aborted  <<< isaac_tutorials [0.89s]
2.888 Aborted  <<< isaacsim [0.90s]
2.893 Aborted  <<< iw_hub_navigation [0.90s]
2.948 Aborted  <<< isaac_ros_navigation_goal [0.97s]
3.902 Aborted  <<< custom_message [1.93s]
4.309 Aborted  <<< isaac_ros2_messages [2.34s]
4.341 
4.341 Summary: 0 packages finished [2.56s]
4.341   1 package failed: cmdvel_to_ackermann
4.341   7 packages aborted: carter_navigation custom_message isaac_ros2_messages isaac_ros_navigation_goal isaac_tutorials isaacsim iw_hub_navigation
4.341   1 package had stderr output: cmdvel_to_ackermann
------
failed to solve: process "/bin/sh -c cd /tmp/ &&     curl -L -O https://github.com/isaac-sim/IsaacSim-ros_workspaces/archive/main.zip &&     unzip main.zip &&     mv IsaacSim-ros_workspaces-main/humble_ws /humble_ws &&     cd /humble_ws &&     . /opt/ros/humble/setup.sh &&     colcon build --symlink-install &&     rm -rf /tmp/IsaacSim-ros_workspaces-main" did not complete successfully: exit code: 1
```

--- a/.env
+++ b/.env
@@ -1,5 +1,5 @@
 PROJECT_NAME="airstack"
-PROJECT_VERSION="1.0.4"
+PROJECT_VERSION="1.0.5"
 # can replace with your docker hub username
 PROJECT_DOCKER_REGISTRY="airlab-storage.andrew.cmu.edu:5001/shared"
 DEFAULT_ISAAC_SCENE="omniverse://airlab-storage.andrew.cmu.edu:8443/Projects/AirStack/fire_academy.scene.usd"

--- a/simulation/isaac-sim/docker/Dockerfile.isaac-ros
+++ b/simulation/isaac-sim/docker/Dockerfile.isaac-ros
@@ -51,7 +51,8 @@ RUN apt update && apt install -y --no-install-recommends curl emacs vim nano tmu
     python3-pip \
     python3-rosdep \
     ros-humble-tf2* \
-    ros-humble-mavros
+    ros-humble-mavros \
+    ros-humble-ackermann-msgs
 
 RUN /opt/ros/humble/lib/mavros/install_geographiclib_datasets.sh
 

